docs/user/accumulo/install.rst

```diff
@@ -44,8 +44,8 @@ Installing the Accumulo Distributed Runtime Library
 ---------------------------------------------------
 
 The ``geomesa-accumulo_2.11-$VERSION/dist/accumulo/`` directory contains the distributed
-runtime JARs that contains server-side code for Accumulo that must be made
-available on each of the Accumulo tablet servers in the cluster. These JARs
+runtime JAR that contains server-side code for Accumulo that must be made
+available on each of the Accumulo tablet servers in the cluster. This JAR
 contain GeoMesa code and the Accumulo iterators required for querying GeoMesa data.
```

Comment:
This JAR *contains*...

(Just proving that I read:))

Reply:
fixed, thanks